app/config/tstvhq/events.json

```diff
@@ -1,84 +1,56 @@
 {
     "events": [
         {
-            "name": "Dark Arias",
-            "description": "Darker arias and art songs sung by a Friends of TST Delaware member (MarissaYoung.com)",
-            "room": "teaRoom",
-            "start": "2020-08-20T20:30:00-04:00",
-            "end": "2020-08-20T21:30:00-04:00",
-            "image": "https://tstvhq.s3.amazonaws.com/events/front parlor movie.jpg"
-        },
-        {
-            "name": "Salem Witch Lecture",
-            "description": "Learn about the Salem Witch Trials from our own Salem Witch, Thomas Vallor (@SalemWitchWiles). This lecture will take place every 30 minutes or until he smokes too much weed.",
+            "name": "Salem Walking Tours",
+            "description": "Salem's history from a Satanic perspective by local Satanic Salem witch Thomas Vallor. This is an hour long presentation.",
             "room": "backParlor",
-            "start": "2020-08-20T19:00:00-04:00",
-            "end": "2020-08-20T01:00:00-04:00",
+            "start": "2020-10-10T02:00:00-04:00",
+            "end": "2020-10-10T03:00:00-04:00",
             "image": "https://tstvhq.s3.amazonaws.com/events/back parlor lecture.jpg"
         },
         {
-            "name": "Sadie's Toy Box",
-            "description": "Educational sex toy presentation by @SadieSatanas",
-            "room": "boudoir",
-            "start": "2020-08-20T20:00:00-04:00",
-            "end": "2020-08-21T02:00:00-04:00",
-            "image": "https://tstvhq.s3.amazonaws.com/events/cooking.jpg"
+            "name": "Satanic Dungeons & Dragons",
+            "description": "DM: Christopher Robichaud ... Friends and members of the Satanic Temple play the popular version of Dungeons & Dragons in the 1980s--First Edition D&D--in the way that everyone was afraid the game was being played back then. Players will get a chance to enjoy an expanded version of the Judges Guild classic old-school module, Inferno (by Geoffry O. Dale), in which characters must descend into the depths of Dante's Hell. But this time around, it's because they're on a rescue mission to release the Dark Prince from his frozen prison.",
+            "room": "crematorium",
+            "start": "2020-10-10T00:00:00-04:00",
+            "end": "2020-10-10T04:00:00-04:00",
```

Comment:
Btw, these are the event times for Saturday, 10/10. I went ahead and changed them to 10/09, but we'll have to update this again Saturday morning.